lib/Ama/Model/Questions.pm

```diff
@@ -90,7 +90,7 @@ sub getQuestions {
   }
   
   if($keyword ne 'none') {
-    $sql = $sql . "question like '%" . $keyword . "%' and ";
+    $sql = $sql . "LOWER(question) like LOWER('%" . $keyword . "%') and ";
```

Comment:
This looks like an SQL injection point.  Using DBI's ? feature for auto-quoting.